presto-docs/src/main/sphinx/presto_cpp/limitations.rst

```diff
@@ -0,0 +1,44 @@
+======================
+Presto C++ Limitations
+======================
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+General Limitations
+===================
+
+There are a number of limitations associated with the C++ evaluation engine.
+
+* Not all built-in functions are implemented in C++. Attempting to use unimplemented functions results in a query failure. For supported functions, see `Function Coverage <https://facebookincubator.github.io/velox/functions/presto/coverage.html>`_.
+
+* Not all built-in types are implemented in C++. Attempting to use unimplemented types will result in a query failure. The list of supported types is **HERE**.
```

Comment:
Where is here?

Reply:
I hadn't received a link for this by the time I needed to open the PR so this was a placeholder. Deleted the sentence entirely and will add it back if I get a link.